RELEASE.md

```diff
@@ -93,7 +93,7 @@ To release a new version of Spyder you need to follow these steps:
 
 * In case a new version is not part of the `defaults` channel yet, you need to copy it to our channel with the following command:
```

Comment:
This is not necessary anymore because we switched to run our tests with Conda-forge. So please remove this bullet point.